build/template_kql/networking/networking_afd_resources_details.kql

```diff
@@ -1,3 +1,3 @@
 ${extend_resource}
 | project Type, SkuName, Kind, SubscriptionId, Name, State, Location, ResourceGroup, Zones, AvailabilityZone, AvailabilitySet, Capacity, FaultDomain, OSDisk, ReadReplica, AutomaticFailover, MultipleWriteLocations, StorageAutogrow, ReplicationRole, Backup = tolower(Backup), Environment, ResourceId
-| where Type == 'microsoft.network/frontdoors'
\ No newline at end of file
+| where Type == 'microsoft.network/frontdoors' or Type == 'microsoft.cdn/profiles'
```

Comment:
Can we add the Json title to match both Front door & CDN?
![image](https://user-images.githubusercontent.com/11145865/208830399-d2b9a192-41d5-470c-a393-dfcc66342add.png)

"Front Door & CDN profiles" as per Azure portal category

Reply:
I agree with your idea. For further detail, please see following new commit.
https://github.com/tsubasaxZZZ/reliability-workbook/commit/5ff71d2bd19233d367871dc4ad80a2607952cf63